src/components/ec-btn/ec-btn.story.js

```diff
@@ -0,0 +1,163 @@
+import { storiesOf } from '@storybook/vue';
+import { boolean, text, select } from '@storybook/addon-knobs';
+import StoryRouter from 'storybook-vue-router';
+import EcBtn from './ec-btn.vue';
+
+const stories = storiesOf('Button component', module).addDecorator(StoryRouter());
+
+stories
+  .add('basic', () => ({
+    components: { EcBtn },
+    props: {
+      roundedFromProps: {
```

Comment:
You need `FromProps` suffix only if the value you are passing to the component is v-model. Component cannot change value of any of these props. You could name them just `rounded`, `outline`, 'fullWidth' and then use it in the template:
```
<ec-btn v-bind="$props" ... />
```